vms/registration/views.py

```diff
@@ -163,19 +167,6 @@ def signup_volunteer(request):
                 return render(request, 'registration/signup_volunteer.html',
                               {'user_form': user_form,
                                'volunteer_form': volunteer_form,
-                               'registered': registered,
-                               'phone_error': phone_error,
-                               'organization_list': organization_list, })
-        else:
-            user_form = UserForm(prefix="usr")
-            volunteer_form = VolunteerForm(prefix="vol")
-
-        return render(request, 'registration/signup_volunteer.html',
-                      {'user_form': user_form,
-                       'volunteer_form': volunteer_form,
-                       'registered': registered,
-                       'phone_error': phone_error,
-                       'organization_list': organization_list, })
-
-    else:
-        return render(request, 'home/home.html', {'error': True})
```

Comment:
@tapasweni-pathak This line is incorporated in the code. ie, in def post() function.